service/controller/v14/version_bundle.go

```diff
@@ -1,64 +1,14 @@
-package v13
+package v14
 
 import (
 	"github.com/giantswarm/versionbundle"
 )
 
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
-		Changelogs: []versionbundle.Changelog{
-			{
-				Component:   "kubernetes",
-				Description: "Updated to 1.10.4.",
-				Kind:        versionbundle.KindChanged,
-			},
-			{
-				Component:   "cloudconfig",
-				Description: "Removed kube-state-metrics and node-exporter related components (will be managed by chart-operator).",
-				Kind:        versionbundle.KindRemoved,
-			},
-			{
-				Component:   "kvm-operator",
-				Description: "Added CA public key into trusted user keys for SSO ssh.",
-				Kind:        versionbundle.KindAdded,
-			},
-			{
-				Component:   "k8s-kvm",
-				Description: "Added virtio RNG device for better entropy generation.",
-				Kind:        versionbundle.KindAdded,
-			},
-		},
-		Components: []versionbundle.Component{
-			{
-				Name:    "calico",
-				Version: "3.0.5",
-			},
-			{
-				Name:    "containerlinux",
-				Version: "1688.5.3",
-			},
-			{
-				Name:    "docker",
-				Version: "17.12.1",
-			},
-			{
-				Name:    "etcd",
-				Version: "3.3.3",
-			},
-			{
-				Name:    "coredns",
-				Version: "1.1.1",
-			},
-			{
-				Name:    "kubernetes",
-				Version: "1.10.4",
-			},
-			{
-				Name:    "nginx-ingress-controller",
-				Version: "0.12.0",
-			},
-		},
```

Comment:
Components can stay. Changelogs should have one dummy item so you can fill it right away.